### PR TITLE
docker_swarm_service: Add stop_grace_period option

### PR DESCRIPTION
**Pull request by hannseman**
##### SUMMARY

Add `stop_grace_period` option.

This PR should bring us in line with all options available in `docker service create/update`.

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

docker_swarm_service

### CODE REVIEW

**Review by jwitko — APPROVED** [review]
LGTM once CI tests are passing

File: lib/ansible/modules/cloud/docker/docker_swarm_service.py [hunk 1]
```diff
@@ -333,6 +333,13 @@
       - List of the service networks names.
       - Corresponds to the C(--network) option of C(docker service create).
     type: list
+  stop_grace_period:
+    description:
+        - Time to wait before force killing a container.
+        - "Accepts a duration as a string in a format that look like:
+          C(5h34m56s), C(1m30s) etc. The supported units are C(us), C(ms), C(s), C(m) and C(h)."
+        - Corresponds to the C(--stop-grace-period) option of C(docker service create).
+    type: str
```
Comment (jwitko) [inline lib/ansible/modules/cloud/docker/docker_swarm_service.py:347]: CI is failing here,  you need `version_added: "2.8"`

Comment (felixfontein) [inline lib/ansible/modules/cloud/docker/docker_swarm_service.py:347]: There needs to be a `version_added` here (after this line). Besides that, looks good!

Comment (felixfontein) [reply lib/ansible/modules/cloud/docker/docker_swarm_service.py:347]: You beat me by 30 seconds ;-)

**Review by felixfontein — APPROVED** [review]
shipit

**Comment by jwitko** [comment]
shipit

**Comment by gundalow** [comment]
@hannseman when you get a chance could you please rebase, thanks in advance.

**Comment by hannseman** [comment]
ready_for_review

**Comment by jwitko** [comment]
shipit

**Comment by gundalow** [comment]
@hannseman Thank you for this feature
@jwitko @felixfontein Thank you for the reviews
Merged into `devel` for release in Ansible 2.8

**Comment by felixfontein** [comment]
@hannseman @jwitko @gundalow thanks for this one, too! :)

**Pull request merged**